osu.Game/Overlays/SettingsOverlay.cs

```diff
@@ -93,7 +96,11 @@ private void load(OsuGame game)
                         new SidebarButton
                         {
                             Section = section,
-                            Action = b => sectionsContainer.ScrollContainer.ScrollTo(b),
+                            Action = s =>
+                            {
+                                sectionsContainer.ScrollTo(s);
+                                sidebar.State = ExpandedState.Contracted;

```

Comment:
you're looking for a setting, the idea is you move off it within 750ms or you likely chose the wrong section and want to see the text again